qa-tests-backend/src/test/groovy/ImageScanningTest.groovy

```diff
@@ -810,8 +810,11 @@ class ImageScanningTest extends BaseSpecification {
             imageDigest = images.find { it.name == imageName }
             assert imageDigest?.id, "image ${imageName} not found among ${images*.name}"
         }
-        ImageOuterClass.Image imageDetail = ImageService.getImage(imageDigest?.id)
-        validateImageMetadata(imageDetail, source)
+        ImageOuterClass.Image imageDetail
+        withRetry(10, 20) {
```

Comment:
Isn't this pretty coarse in terms of retries?

Reply:
just thinking that in a failure case this is 200s so 3.5 minutes waiting on a test failure

Reply:
I'd like to give it a good chance to pass if *CR are slow. In the previous incarnation it would be 5-10 minutes between the first exec and the retry (due to debug gathering).